lib/css/z.thumbviewer.css

```diff
@@ -0,0 +1,42 @@
+/**
+ * css for the thumbviewer
+ */
+
+span.thumbviewer.overlay {
+    color: white;
+    cursor: pointer;
+    z-index: 8000;
+    font-size: xx-large;
+    text-align: right;
+    padding-top: 10px;
+    padding-right: 10px;
+}
+
+span.thumbviewer.overlay:before {
+    content: "⊗"
```

Comment:
Might be good to specify a font here, for safety.

Reply:
Like this? Helvetica, Arial, Sans-serif?